### PR TITLE
Gaia version bump

### PR DESCRIPTION
**Pull request by faddat**

### CODE REVIEW

**Comment by tombeynon** [comment]
Hey @faddat, thanks for the submission. 

First off - is Gaia V5 in production yet? [The release notes](https://github.com/cosmos/gaia/releases/tag/v5.0.0) suggest it's a pre-release. It might still be useful in Omnibus but at the moment we're only setup to build a single version of each chain so I want to keep it as 'current production version'. 

Secondly if we were to merge this, there's a few more files that need updating. See the recent Kava update PR for a good example of what files need changing - https://github.com/ovrclk/cosmos-omnibus/pull/9/files

**Comment by faddat** [comment]
Tom, Gaia v5 is in production.

**Comment by faddat** [comment]
Screen Shot 2021-08-09 at 5 42 53 AM

**Review by nullmames — COMMENTED** [review]
version mismatch with gaia/deploy.yml

File: gaia/docker-compose.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ services:
         PROJECT: gaia
         PROJECT_BIN: gaiad
         PROJECT_DIR: .gaia
-        VERSION: v4.2.1
+        VERSION: v5.0.2
```
Comment (nullmames) [inline gaia/docker-compose.yml:11]: update to match gaia/deploy.yml

**Comment by tombeynon** [comment]
@faddat sorry for the delay - I've just tried running both v5.0.2 and v5.0.5 locally and neither pass the `validate-genesis` step. It passes fine with v4.2.1. Is that expected? 

The error returned is `error validating genesis file /root/.gaia/config/genesis.json: failed to unmarshal liquidity genesis state: EOF`. The genesis.json is from `cosmos/mainnet`.

**Comment by tombeynon** [comment]
Closing as superseded by #32

**Pull request closed without merging**